domaci22.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+<link rel="stylesheet" href="stil22.css"> 
+
+</head>
+
+<body>  <div class= "container">
+    
+    
+      <div class = "split">
+        <h1 class="grad"> Život u Splitu </h1>  
```

Comment:
Class `.grad` can not tell other CSS developer what is represents without checking it visually. `.title` might be better int this case.